queue_job/delay.py

```diff
@@ -534,6 +534,45 @@ def delay(self):
         """Delay the whole graph"""
         self._graph.delay()
 
+    def split(self, size=None):
+        """Split the Delayable into a DelayableGroup containing batches
+        of size `size`
+        """
+
+        total_records = len(self.recordset)
+        count = 1 + total_records // size
+
+        delayables = []
+        for batch in range(count):
```

Comment:
Perhaps simpler to let `range` do more of the work?

```python
for start in range(0, total_records, size):
```

Reply:
Good idea